packages/pds/src/db/tables/repo-op.ts

```diff
@@ -0,0 +1,11 @@
+export interface RepoOp {
+  did: string
+  commit: string
+  action: string
```

Comment:
Whether or not it ends-up in the lex schema, we could type this as `WriteOpAction`.